workflow-testing/src/test/java/com/squareup/workflow1/testing/RealRenderTesterTest.kt

```diff
@@ -1025,6 +1026,100 @@ internal class RealRenderTesterTest {
     }
   }
 
+  @Test fun `verifyActionOutput throws when no output`() {
+    val workflow = Workflow.stateless<Unit, Nothing, Sink<TestAction>> {
+      actionSink.contraMap { it }
+    }
+    val testResult = workflow.testRender(Unit)
+      .render()
+    assertFails {
```

Comment:
lets tighten this up to `assertFailsWith(IllegalStateException("Expected an output but no output was found."))`

Reply:
I updated this one to an AssertionError instead of IllegalStateException since all there were other test method which were throwing AssertionErrors.